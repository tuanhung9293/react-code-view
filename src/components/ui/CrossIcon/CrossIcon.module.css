.base {
  width: 20px;
  min-width: 15px;
  height: auto;
  max-width: 100%;
  display: block;
}

.small {
  width: 15px;
  min-width: 15px;
}