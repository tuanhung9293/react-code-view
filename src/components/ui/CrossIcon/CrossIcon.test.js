import React from 'react'
import {shallow} from 'enzyme'
import CrossIcon from './'

it('renders CrossIcon with no props', () => {
  expect(shallow(
    <CrossIcon />
  )).toMatchSnapshot()
})

it('renders CrossIcon with props dark', () => {
  expect(shallow(
    <CrossIcon dark />
  )).toMatchSnapshot()
})

it('renders CrossIcon with props dark', () => {
  expect(shallow(
    <CrossIcon small />
  )).toMatchSnapshot()
})

