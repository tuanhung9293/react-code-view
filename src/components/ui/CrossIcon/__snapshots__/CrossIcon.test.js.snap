// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CrossIcon with no props 1`] = `
<svg
  className="base"
  viewBox="0 0 20 20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M629.854051,297.732731 L637.465462,290.12132 L635.344142,288 L627.732731,295.611411 L620.12132,288 L618,290.12132 L625.611411,297.732731 L618,305.344142 L620.12132,307.465462 L627.732731,299.854051 L635.344142,307.465462 L637.465462,305.344142 L629.854051,297.732731 Z"
    fill="#ffffff"
    transform="translate(-618 -288)"
  />
</svg>
`;

exports[`renders CrossIcon with props dark 1`] = `
<svg
  className="base"
  viewBox="0 0 20 20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M629.854051,297.732731 L637.465462,290.12132 L635.344142,288 L627.732731,295.611411 L620.12132,288 L618,290.12132 L625.611411,297.732731 L618,305.344142 L620.12132,307.465462 L627.732731,299.854051 L635.344142,307.465462 L637.465462,305.344142 L629.854051,297.732731 Z"
    fill="#000000"
    transform="translate(-618 -288)"
  />
</svg>
`;

exports[`renders CrossIcon with props dark 2`] = `
<svg
  className="base small"
  viewBox="0 0 20 20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M629.854051,297.732731 L637.465462,290.12132 L635.344142,288 L627.732731,295.611411 L620.12132,288 L618,290.12132 L625.611411,297.732731 L618,305.344142 L620.12132,307.465462 L627.732731,299.854051 L635.344142,307.465462 L637.465462,305.344142 L629.854051,297.732731 Z"
    fill="#ffffff"
    transform="translate(-618 -288)"
  />
</svg>
`;
